client/web/compose/src/components/PageBlocks/Nylas/NylasConfigurator.vue

```diff
@@ -16,9 +16,114 @@
         v-model="options.componentID"
       />
     </b-form-group>
+
+    <template v-if="viewPrefillSection">
```

Comment:
showPreviewSection is a better description for the computed
view indicates something will happen, so it would usually be a method name or similar.